### PR TITLE
Metrics page

### PR DESCRIPTION
**Pull request by mpolimen**

### CODE REVIEW

**Comment by GraceMattern** [comment]
Looks good overall but I wasn't able to view it with dummy data. What is the cycle that is linked with the dummy data? Also can you center or make the loading text bigger

**Comment by mpolimen** [comment]
@GraceMattern the dummy data is now fully removed from the branch

**Comment by mpolimen** [comment]
As for the loading text, I was thinking of maybe making a card to do a site-wide overhaul for how we show errors and loading outside of a div with text

**Comment by mpolimen** [comment]
@roccopolimen will review this, but I am making a quick change for Brian, so don't merge until my next push

File: client/src/App/index.tsx [hunk 1]
```diff
@@ -29,6 +30,7 @@ const queryClient: QueryClient = new QueryClient({
 });
 
 const App = () => {
+    document.body.style.backgroundColor = "#F9F8FF";
```
Comment (roccopolimen) [inline client/src/App/index.tsx]: we are using MUI for theming not this

**Review by roccopolimen — COMMENTED** [review]
One comment, I'll fix this myself

**Pull request merged**